### PR TITLE
Fix mqa is false case in gpt_bigcode

### PR DESCRIPTION
**Pull request by zhaoyang-star**
cc @zhuohan123

### CODE REVIEW

File: vllm/model_executor/models/gpt_bigcode.py [hunk 1]
```diff
@@ -49,10 +49,11 @@ def __init__(self, config: GPTBigCodeConfig):
...
         self.hidden_size = config.hidden_size
         total_num_heads = config.num_attention_heads
-        tensor_model_parallel_world_size = (
+        self.tensor_model_parallel_world_size = (
             get_tensor_model_parallel_world_size())
-        assert total_num_heads % tensor_model_parallel_world_size == 0
-        self.num_heads = total_num_heads // tensor_model_parallel_world_size
+        assert total_num_heads % self.tensor_model_parallel_world_size == 0
+        self.num_heads = \
+            total_num_heads // self.tensor_model_parallel_world_size
```
Comment (zhuohan123) [inline vllm/model_executor/models/gpt_bigcode.py:56]: Nit: Please use parenthesis for line continuation according to google python style guide.
```suggestion
        self.num_heads = (
            total_num_heads // self.tensor_model_parallel_world_size)
```

File: vllm/model_executor/models/gpt_bigcode.py [hunk 2]
```diff
@@ -286,7 +288,8 @@ def load_weights(self,
                 hidden_size = self.config.hidden_size
                 head_size = hidden_size // total_num_heads
                 total_kv_size = head_size * total_num_kv_heads
-                num_heads = total_num_heads // tensor_model_parallel_world_size
+                num_heads = \
+                    total_num_heads // self.tensor_model_parallel_world_size
```
Comment (zhuohan123) [inline vllm/model_executor/models/gpt_bigcode.py:292]: Ditto on line continuation

**Review by zhuohan123 — CHANGES_REQUESTED** [review]
Thanks for the fix! Left some small comments on style.

**Comment by zhaoyang-star** [comment]
> Thanks for the fix! Left some small comments on style.

Fixed. Please have a review again. @zhuohan123 
PR about QKV gemm fuse in gpt_bigcode will be committed later.

**Review by zhuohan123 — APPROVED** [review]
LGTM! Thank you for your contribution!

**Pull request merged**